pkg/route/api/v1/conversion.go

```diff
@@ -10,8 +10,16 @@ import (
 func addConversionFuncs(scheme *runtime.Scheme) {
 	err := scheme.AddDefaultingFuncs(
 		func(obj *RouteSpec) {
-			if len(obj.To.Kind) == 0 {
-				obj.To.Kind = "Service"
+			if obj.AdditionalTos == nil {
```

Comment:
This should not be necessary - we generally don't do this sort of defaulting.  A nil slice is the same as an empty slice.